File: src/main/java/com/anthonymariotti/ticketmaster/utilities/TicketMasterLogger.java
package com.anthonymariotti.ticketmaster.utilities;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class TicketMasterLogger {
    private static Logger LOGGER = LoggerFactory.getLogger("TicketMaster");

    public static void debug(String message) {
        LOGGER.debug(message);
    }

    public static void info(String message) {
        LOGGER.info(message);
    }

    public  static void error(Throwable e, String message) {
        LOGGER.error(message, e);
    }
}

File: src/main/kotlin/com/anthonymariotti/ticketmaster/utilities/TicketMasterLogger.kt
package com.anthonymariotti.ticketmaster.utilities

import org.slf4j.LoggerFactory

private val LOGGER = LoggerFactory.getLogger("TicketMaster")

fun logInfo(message: String) {
    LOGGER.info(message)
}

fun logWarning(message: String) {
    LOGGER.warn(message)
}

fun logError(message: String) {
    LOGGER.error(message)
}

fun logError(message: String, error: Throwable) {
    LOGGER.error(message, error)
}